.github/workflows/build.yml

```
@@ -0,0 +1,45 @@
+name: CI
+
+on:
+  push:
+    branches: [ medic-conf-1.17 ]
+  pull_request:
+    branches: [ medic-conf-1.17]
```

```suggestion
on: [push, pull_request]
```

I think it should be fine to do this for all branches.  The build will not succeed any that do not include these changes we are adding to `medic-conf-1.17`, but that is fine.